src/login/Login.js

```diff
@@ -24,7 +24,7 @@ export default class Login extends React.Component {
         this.errors = json.text
         otpLength = json.otpLength;
       } catch (e) {
-        /* */
+        console.log(`could not pass error json: ${e}`);
```

Comment:
Can't we just write a generic error message to the error div? Like `An unknown error occurred.` or whatever is a good fit here?